### DISCUSSION TITLE
Analysis through Jupyter?

### DISCUSSION THREAD

**Discussion by lb815** [SchmollerLab/Cell_ACDC#550] [General]
Hi, I am very new to CellACDC and have a question regarding the jupyter notebook ("cell_cycle_analysis.ipynb"). I'm mostly going off the quick start guide here, and am not super familiar with code in general. 

When I initially open jupyter lab through the acdc terminal, I can't find the notebook folder where the cell cycle analysis notebook is supposed to be located. I tried just downloading it from the github and opening it that way, but when I try to run the configuration steps I get a error that says:  ModuleNotFoundError: No module named 'cellacdc'. 

Is there something I'm missing in terms of getting the notebook setup to analyze CellACDC data? Thanks so much!

**Comment by ElpadoCan** [comment]
Hi @lb815,

Can you send me a screenshot of the terminal when you type the `jupyter` command?

Did you install Cell-ACDC? Does it work?

Downloading from github is totally fine and it should work but the error you get means that you are launching the notebook from and environment that does not have cellacdc installed, but we can certainly fix that.

Let me know, thanks!

**Comment by lb815** [comment]
Hi, Thanks for responding! Yes, Cell-ACDC is installed and does work. I've included a picture of how I've been trying to open jupyter in Windows Powershell (I've also tried this via Conda on a different computer, which gets the same result). I've also included a picture of the error code in Jupyter. 

![image](https://github.com/user-attachments/assets/907401fd-9ab0-46b7-bf27-bf043515bc74)
![image](https://github.com/user-attachments/assets/caaf1efa-44e1-474d-b85b-c9b0bfbfa9de)

Thanks again!

**Comment by ElpadoCan** [comment]
Oh ok you will need to add the correct kernel then. 

To do this activate the acdc env and then run these commands one by one:
```
python -m pip install --upgrade ipykernel
python -m ipykernel install --user --name=acdc
```

When you run jupyter again you will be able to change the kernel to the acdc environment. You can change the kernel from the top-right drop-down menu like in the screenshots below.

After having selected the correct kernel the notebook should work. Let me know thanks! 

![image](https://github.com/user-attachments/assets/24b14198-2e5d-41bb-84c4-200bc738fee9)
![image](https://github.com/user-attachments/assets/508c2e19-8f00-4989-b19a-6bed6793e744)

**Comment by lb815** [comment]
This worked, thank you so much!
